src/backy/tests/test_archive.py

```diff
@@ -60,25 +61,68 @@ def test_load_revisions(backup_with_revisions):
     assert a.history[2].get_parent().uuid == "123-1"
     assert a.history[0].get_parent() is None
     assert a.find_revisions("all") == a.history
-    assert a.find_revisions(1) == [a.find(1)]
+    assert a.find_revisions("1") == [a.find("1")]
+    assert a.find_revisions("trust:verified") == [a.find("123-0")]
+    assert a.find_revisions("2..1") == [a.find("2"), a.find("1")]
```

Comment:
`1..2 `should also return the right range as that is what people will intuitively use the first time they use this themselves. Keeping it in the defined order of "oldest last, newest first" is fine.